test/integration/test_sampler.py

```diff
@@ -166,3 +166,21 @@ def test_sampler_primitive_as_session(self, service):
             self.assertIsInstance(result, SamplerResult)
             self.assertEqual(len(result.quasi_dists), len(circuits0))
             self.assertEqual(len(result.metadata), len(circuits0))
+
+    @run_integration_test
+    def test_sampler_raise_error(self, service):
```

Comment:
Can you move this test to `unit/test_ibm_primitives.py`? It doesn't need to be an integration test since the check is done locally and integration test is expensive. You can just mock the `Session`.

Reply:
Actually, presumably you have tests in terra, so the test here isn't really necessary as we'll eventually use terra code.

Reply:
OK. I then remove the test.